gobblin-modules/gobblin-kafka-09/src/test/java/org/apache/gobblin/runtime/KafkaAvroJobStatusMonitorTest.java

```diff
@@ -356,7 +357,8 @@ public void testProcessingRetriedForApparentlyTransientErrors() throws IOExcepti
     }, 2, 2, TimeUnit.SECONDS);
     Thread mainThread = Thread.currentThread();
     // guardrail against excessive retries (befitting this unit test):
-    toggleManagementExecutor.scheduleAtFixedRate(mainThread::interrupt, 60, 5, TimeUnit.SECONDS);
+    toggleManagementExecutor.scheduleAtFixedRate(mainThread::interrupt, 20, 5, TimeUnit.SECONDS);
```

Comment:
so is the present EB series - [2s, 4s, 8s, 16s, 20s, 20s, 20s, ... ]?  if so only 20s may not be long enough to allow for 5 "fake exception" failures.  or am I getting my numbers wrong?

Reply:
actually, now I see the 1ms above, so sequence of [1ms, 2ms, 4ms, 8ms, 16ms, 32ms, ... ]

all good!